### PR TITLE
samples: openthread: no need for separated nRF53 cert file

### PR DESCRIPTION
**Pull request by edmont**
After multiprotocol is supported by nRF53 devices, there is no need to have separated Thread Certification overlay file.

### CODE REVIEW

**Review by lmaciejonczyk — APPROVED** [review]

File: doc/nrf/ug_thread_certification.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ Complete the following steps to run the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with an overlay file.
+   The :ref:`ot_cli_sample` sample is used as a base, modified with an overlay file: :file:`harness/overlay-cert.conf`.
```
Comment (ru-fu) [inline doc/nrf/ug_thread_certification.rst:61]:
```suggestion
   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
```

File: doc/nrf/ug_thread_certification.rst [hunk 2]
```diff
@@ -67,13 +67,8 @@ Complete the following steps to run the certification tests:
 
    .. note::
       The overlay file selects the precompiled OpenThread libraries.
-
-      For nRF52 Series devices, use :file:`harness/overlay-cert.conf`.
       This file also enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth LE advertising.
```
Comment (ru-fu) [inline doc/nrf/ug_thread_certification.rst:70]:
```suggestion
      It also enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth LE advertising.
```

**Review by ru-fu — COMMENTED** [review]
A little bit of nitpicking, sorry. ;)

File: doc/nrf/ug_thread_certification.rst [hunk 3]
```diff
@@ -58,21 +58,15 @@ Complete the following steps to run the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with an overlay file.
-
+   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
```
Comment (ru-fu) [inline doc/nrf/ug_thread_certification.rst:61]: You need a blank line after this and before the code-block, or it will not show up correctly.

**Review by ru-fu — APPROVED** [review]

**Review by pszkotak — APPROVED** [review]

**Pull request merged**